### PR TITLE
Added missing outposts to the travel window

### PR DESCRIPTION
**Pull request by quetzalsly**
Works in presearing and postsearing.
If the player has unlocked everything then the combo box should not be drawn at all.
Haven't tested this in game because I don't have a char that has unlocked everything yet.

### CODE REVIEW

File: GWToolboxdll/Windows/TravelWindow.cpp [hunk 1]
```diff
@@ -164,6 +164,53 @@ void TravelWindow::TravelButton(const char* text, const int x_idx, const GW::Con
...
+bool TravelWindow::PlayerHasAnyMissingOutposts(bool presearing) {
+    if (presearing) {
+        if (!GW::Map::GetIsMapUnlocked(GW::Constants::MapID::Ascalon_City_pre_searing))return true;
+        if (!GW::Map::GetIsMapUnlocked(GW::Constants::MapID::Ashford_Abbey_outpost))return true;
+        if (!GW::Map::GetIsMapUnlocked(GW::Constants::MapID::Foibles_Fair_outpost))return true;
+        if (!GW::Map::GetIsMapUnlocked(GW::Constants::MapID::Fort_Ranik_pre_Searing_outpost))return true;
+        if (!GW::Map::GetIsMapUnlocked(GW::Constants::MapID::The_Barradin_Estate_outpost))return true;
+    }
+    else {
+        for (int i = 0; i < (int)outpost_names.size(); ++i) {
```
Comment (franneck94) [inline GWToolboxdll/Windows/TravelWindow.cpp:176]: we want to use c++ cast only please

File: GWToolboxdll/Windows/TravelWindow.h [hunk 2]
```diff
@@ -64,6 +64,8 @@ class TravelWindow : public ToolboxWindow {
     static GW::Constants::MapID IndexToOutpostID(int index);
     static bool ParseDistrict(const std::wstring& s, GW::Constants::District& district, uint32_t& number);
     static bool ParseOutpost(const std::wstring& s, GW::Constants::MapID& outpost, GW::Constants::District& district, const uint32_t& number);
+    bool PlayerHasAnyMissingOutposts(bool presearing);
```
Comment (franneck94) [inline GWToolboxdll/Windows/TravelWindow.h:67]: these params can be const

File: GWToolboxdll/Windows/TravelWindow.cpp (hunk 1 above)
Comment (DubbleClick) [reply GWToolboxdll/Windows/TravelWindow.cpp:176]: preferably no cast and auto i = 0u;

File: GWToolboxdll/Windows/TravelWindow.h (hunk 2 above)
Comment (DubbleClick) [reply GWToolboxdll/Windows/TravelWindow.h]: this is a declaration, they should be const in the definition only

Comment (franneck94) [reply GWToolboxdll/Windows/TravelWindow.h]: Is there a reason not having at also at the declaration?
I see no benefit in doing this

**Pull request merged**